### ISSUE TITLE
Gather ddev download statistics

### ISSUE THREAD

**Issue by cyberswat** [drud/ddev#532]
## What happened (or feature request):
We would like to capture daily download statistics from the github api for ddev for each of the assets associated with a release.

## What you expected to happen:
Create a tool that can parse the assets for a given release and store totals in a cumulative csv file.

The following items walk through workflow for the v0.9.3 release.  I assume that repo and release are arguments passed to the tool:
- [ ] Retrieve/create a target file to append download counts to.
- [ ] Make a GET request to https://api.github.com/repos/drud/ddev/releases/8137672 and parse the resulting json.
- [ ] for each item stored in the "assets" array append a current timestamp, id, name, content_type, size, download_count (only the difference since the previous day), created_at and updated_at fields as a row in a cumulative csv file.  If a row previously exists for the given day then it is overwritten with the latest values.
- [ ] store the csv file in a dedicated cloud location like s3 or as a git commit into a repository.
The github api only stores the latest totals and does not offer a breakdown per day.  Once this tool is created we can have another issue that focuses on getting daily trends.

**Comment by rickmanelius** [comment]
Related https://github.com/drud/ddev/issues/307

**Comment by rickmanelius** [comment]
I had a valiant effort tonight, but ultimately failed on two attempts. With Golang I could get ID associated with the latest release and then use that to get to the json payload containing the asset data, but I'm tripping up on Golang's structs and I'm not patient enough to wade through all the tutorials. I attempted ruby in parallel but there I was having all sorts of issues on trying to get the get request with the appropriate json entry in the header. I'll just manually parse for now and then look to get some additional help from the team to do this correctly.

**Comment by rickmanelius** [comment]
On the plus side, the v1.3.1 release has been downloaded 180 times (as reported by GitHub).

**Comment by rfay** [comment]
Can this be closed @cyberswat ? I know you've done a lot of work in this area.

**Comment by jrthrawny** [comment]
We can. There is a github exporter for prometheus, but I don't see downloads in it. It could be that they just didn't add that metric. I will investigate further.
